### DISCUSSION TITLE
NFTs in Idena based on Proof of Personhood ?

### DISCUSSION THREAD

**Discussion by triorigin** [idena-network/idena-go#872] [Ideas]
Once Smart contracts gets into action, it would be great to have NFTs minted in idena once every 3 validations, the NFTs will be distributed based on the Age and Accuracy of the ID during the last 3 epochs. After validation ceremony new rare NFT items being collected, one could finally mint it in the marketplace. This would encourage identities to participate validation ceremony without missing them and try to reach highest accuracy. 

The NFTs which are minted will have unique value due to Proof of personhood, the highest rarity and traits will be allocated and distributed based on the performance and contribution of Identity.

This is just a general Idea we could enhance it even more better, i believe every Human ID in Idena is very unique and all those Human identities who have been validating and contributing to the network should deserve something big. 
NFT on Idena based on Proof Of Person is a very interesting topic to discuss about.

**Comment by idena-Egyptian-Pharoh** [comment]
Of sell invites in nft shape so ppl can buy it with idna coins
